src/jit/codegeninterface.h

```diff
@@ -518,6 +518,13 @@ class CodeGenInterface
         siVarLoc(const LclVarDsc* varDsc, regNumber baseReg, int offset, bool isFramePointerUsed);
         siVarLoc(){};
 
+        // Live this commented because there is already an Equals definition below
```

Comment:
I do not understand this comment and the following functions, do we need them?

Reply:
I was thinking on how to avoid next developers overloading operator==/!=. Do you think I should remove it?

Reply:
I think such overloading is already pretty rare in JIT. The couple of cases you found in a search above seem to be mostly STL style iterators where that's normal but otherwise a rarity in JIT.

Reply:
I will remove it. Thanks!